cypress/e2e/gui/homepage_gui_tests.cy.js

```diff
@@ -0,0 +1,37 @@
+
+describe('Homepage GUI validations', () => {
+
+    // Constants used across multiple tests
+    const today = new Date(2024, 6, 9) // July 9, 2024
+    const formattedDate = today.toDateString()
+
+    beforeEach(() => {
+        cy.visit('/')
+        cy.clock(today.getTime())
+    })
+
+    it('Homepage renders the header with an h1 and theme toggle*', () => {
+
+        cy.contains('h1', 'EngageSphere')
+            .should('be.visible')
+
+        cy.get('#theme-toggle-button')
+            .should('be.visible')
+    })
```

Comment:
It's a good practice to leave an empty line between test cases to improve readability.

```suggestion
    })
    
```

Reply:
Sorry I totally missed that, as I always do leave a space in between tests. Fixed!